features/markdown.feature

```diff
@@ -0,0 +1,7 @@
+Feature: Markdown entries
+    Entries should display as markdown objects.
+
+    Scenario: Entry is markdown
+        Given an entry with the title "Markdown Test"
+        When I see the entry on the index page
+        Then I see that it is a markdown entry
```

Comment:
In general, try to put more than one scenario into a features file.  You can think of the "features" as being "An edit page" and "A detail page".  From there, scenarios test the various aspects of each of those two.  So I think you'd do better to have just two feature files so far.